packages/flutter_tools/test/daemon_test.dart

```
@@ -126,7 +113,7 @@ void main() {
       });
     });
 
-    _testUsingContext('daemon.start', () async {
+    testUsingContext('daemon.start', () async {
```

"app.start without a deviceId should report an error"